src/support/variables/layout.scss

```diff
@@ -40,25 +40,25 @@ $spacer-6: $spacer * 5 !default; // 40px
 
 // The list of spacer values
 $spacers: (
-  $spacer-0,
-  $spacer-1,
-  $spacer-2,
-  $spacer-3,
-  $spacer-4,
-  $spacer-5,
-  $spacer-6,
+  0,
+  var(--base-size-4),
```

Comment:
I think this file should probably stay the same right? Otherwise I think we're removing the spacer variables, which would probably be a breaking change